src/NosCore.GameObject/Providers/MinilandProvider/MinilandProvider.cs

```diff
@@ -1,4 +1,4 @@
-﻿//  __  _  __    __   ___ __  ___ ___
+//  __  _  __    __   ___ __  ___ ___
```

Comment:
?

Reply:
I don't know why this get changed, It's the first line with ASCI NOSCORE.